lib/internal/modules/cjs/loader.js

```diff
@@ -1144,7 +1144,7 @@ function getDefaultLoad(url, filename) {
   return function defaultLoad(urlFromHook, context) {
     // If the url is the same as the original one, save the conversion.
     const isLoadingOriginalModule = (urlFromHook === url);
-    const filenameFromHook = isLoadingOriginalModule ? filename : convertURLToCJSFilename(url);
+    const filenameFromHook = isLoadingOriginalModule ? filename : convertURLToCJSFilename(urlFromHook);
```

Comment:
How could this matter if `url` and `urlFromHook` are the same?

Reply:
They aren't the same (`isLoadingOriginalModule` condition is `false`, so they aren't the same in that branch). Without this change, we cannot change the URL in the load hook for the next hook in the chain.

Reply:
OH! You're right-it being condensed onto 1 line made it difficult to read. Thanks!